README-CrossDac.md

```diff
@@ -0,0 +1,62 @@
+### Cross OS DAC Readme
+
+This repo is a fork of the primary CoreCLR release/3.1 branch.
+
+It exists to implement the Windows version of the DAC & DBI libraries
+which we use for debugging. It enables examination of Linux core dumps
```

Comment:
```suggestion
which we use for debugging and enables examination of Linux core dumps
```

Reply:
Feels like this introduces a run-on sentence.